uisoup/win_soup/mouse.py

```diff
@@ -179,7 +179,7 @@ def click(self, x, y, button_name=LEFT_BUTTON):
             self._compose_mouse_event(button_name, press=True, release=True),
             0, 0, 0, 0)
 
-    def double_click(self, x, y, button_name=LEFT_BUTTON):
+    def double_click(self, x, y, button_name=LEFT_BUTTON, dbl_time=0.5):
```

Comment:
>I suppose comment will help.
do you suppose the noble user of that venerable library to open source code, use provided link and read that fucking wall of text from msdn to understand what really that parameter means? you joking, right?